content/projects/vb-dotnet/manipulate-virtual-keyboard.md

```diff
@@ -0,0 +1,23 @@
+---
+title: ติดต่อและควบคุมแป้นคีย์บอร์ดเสมือน (Virtual Keyboard)
+showMetadata: true
+editable: true
+showToc: true
+categories:
+  - project
+tags:
+  - WinAPI32
+---
+
+Original Author: [Suraphon Yotsanga](https://www.facebook.com/g2gnet)
```

Comment:
Should we start using author field in frontmatter?

Reply:
I will add later, because now we don't use that info to render any UI.

Reply:
@mildronize added an issue https://github.com/dotnetthailand/dotnetthailand.github.io/issues/134